getting_started/step_by_step/signals.rst

```diff
@@ -300,6 +300,8 @@ We can now connect the Timer to the Sprite2D in the ``_ready()`` function.
     func _ready():
         var timer = get_node("Timer")
         timer.timeout.connect(_on_timer_timeout)
+        #for version 4.x use
+        #timer.connect("timeout", _on_timer_timeout)
```

Comment:
This is incorrect, the correct syntax is the above, and this *is* for 4.x, so this change doesn't make sense

Reply:
Hi @AThousandShips , to me the line 302 doesn't works, an alternative code I used was the line 304 to godot version 4.1.3,  be free to change anything you want, thank you.

Reply:
Tested and yes the code works, so likely an error in copying the code

This should be closed

Reply:
Have you confirmed you did not make this mistake?